src/Modules/Factory/Config/RemoveUnusedNodes.module

```diff
@@ -0,0 +1,19 @@
+{
+  "module": {
+    "name": "RemoveUnusedNodes",
+    "namespace": "Fields",
+    "status": "In progress: needs more testing",
+    "description": "Removes meshnodes that are not connected in any mesh
```

Comment:
@moritzdannhauer This is breaking the build. Newlines in these descriptions are not supported.